res/values/strings.xml

```diff
@@ -543,7 +566,10 @@
     <string name="profile_image_select">Select profile image</string>
     <string name="select_your_new_profile_image">Select your new profile image</string>
     <string name="profile_image_delete">Delete profile image</string>
-
+    <string name="pref_show_tray_icon">Show tray icon</string>
+    <string name="pref_edit_profile">Edit profile</string>
+    <string name="pref_set_profile_photo">Set profile photo</string>
+    <string name="pref_remove_profile_photo">Remove profile photo</string>
```

Comment:
the "profile photo" is called "profile image" everywhere else; we should keep this term.

so, please use the already existing string `profile_image_delete` that reads "Delete profile image".

```suggestion
```